convene-web/config/locales/en.yml

```diff
@@ -48,10 +48,11 @@ en:
     sessions:
       create:
         session_expired: 'Your session has expired, please sign in again.'
-        email_sent_if_record_found: "We've sent you an email."
+        email_sent_if_record_found: "Check your email for a link to log in automatically."
         token_claimed: "This link has already been used, try requesting the link again"
       new:
-        submit: 'Send magic link'
+        email_label: 'Your Email'
+        submit: 'Submit'
```

Comment:
Few things make me happier than using I18n for _every single_ piece of text we show people <3 <3 <3